data/tools/wasabi.yml

```diff
@@ -0,0 +1,11 @@
+name: Wasabi
+categories:
+  - linter
+tags:
+  - webassembly
+license: Unknown
```

Comment:
This should be `MIT License`. See https://github.com/danleh/wasabi/blob/master/LICENSE